apps/cricket-score-frontend/src/components/match/TeamScore.tsx

```
@@ -0,0 +1,66 @@
+          <p className="text-xl font-bold text-foreground">
+            {team.name}
+            {team.isBatting && (
+              <span className="badge-soft ml-2 uppercase tracking-wider text-red-400 border-red-500/30 bg-red-500/10">
+                Batting
+              </span>
+            )}
+          </p>
+          {team.overs !== undefined && (
+            <p className="text-sm font-medium text-muted-foreground">
+              <span className="inline-block mr-1">
+                <svg xmlns="http://www.w3.org/2000/svg" className="h-4 w-4 inline" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)